app/javascript/packages/document-capture/components/po-search-fallback-select.scss

```diff
@@ -0,0 +1,3 @@
+.usa-input--error.usa-input--error.usa-input--error {
+  border-color: #e21c3d;
```

Comment:
Not a blocker, but would like to understand why this is needed. Is the error border not appearing correctly for select fields by default without the extra specificity in the selector?

This would also apply to any input on the document capture step, though I don't know if there's actually anything in practice that would be affected by this currently.

Reply:
We noticed that the red border wasn't being displayed on error because the `usa-input--error` border color was being overridden by `usa-select`'s border color. It wasn't happening on another of our pages with a select because the inclusion of the memorable date component on the same page caused multiple declarations of `usa-input--error`. This change reproduces that scenario for this implementation of the select.